public/views/viewBuildError.html

```diff
@@ -9,6 +9,18 @@
     <link rel="stylesheet" type="text/css" href="css/jquery-ui-1.10.4.css" />
     <link rel="stylesheet" type="text/css" href="css/jqModal.css" />
     <link rel="stylesheet" type="text/css" ng-href="{{cssfile}}" />
+    <link rel="stylesheet" href="css/bootstrap.min.css"/>
+    <style type="text/css">
```

Comment:
You should probably make these classes and tag the appropriate elements. Preferably this would be in the CSS file.